classes/autoptimizeToolbar.php

```diff
@@ -0,0 +1,127 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+class autoptimizeToolbar {
+
+	public function __construct()
+	{
+		// If Cache is not available we don't add the Autoptimize Toolbar
+		if( !autoptimizeCache::cacheavail() ) return;
+
+		// Load admin toolbar feature once WordPress, all plugins, and the theme are fully loaded and instantiated.
+		add_action( 'wp_loaded', array( $this, 'load_toolbar' ) );
+	}
+
+	public function load_toolbar()
+	{
+		// We check that the current user has the appropriate permissions
+		if( current_user_can( 'manage_options' ) )
```

Comment:
can you make that;
`
if( current_user_can( 'manage_options' ) && apply_filters('autoptimize_filter_toolbar_show',true) )
`

to allow (technical) users to disable the toolbar from being shown?